sos/report/__init__.py

```diff
@@ -736,6 +736,18 @@ def _add_sos_logs(self):
             self.archive.add_file(self.sos_ui_log_file,
                                   dest=os.path.join('sos_logs', 'ui.log'))
 
+    def _obfuscate_upload_passwords(self):
+        # obfuscate strings like:
+        # --upload-pass=PASSWORD
+        # --upload-pass PASSWORD
+        # --upload-url https://user:PASSWORD@some.url
+        # in both sos_logs/sos.log and in sos_reports/manifest.json
+        for logfile in [os.path.join('sos_logs', 'sos.log'),
+                        os.path.join('sos_reports', 'manifest.json')]:
+            for re in [r"(upload-pass[\s=]+)\S+",
+                       r"(--upload-url[\s=]+\S+://.*:)([^@]*)"]:
```

Comment:
Just for my knowledge, what is the reason for the difference in syntax between `upload-pass` and `--upload-url`?

Reply:
no reason - I can (and should) add `--` before `upload-pass`, good point.